_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
 
-## Istio 0.3.0 features status
+## Istio 0.4 features
 
 Below is our list of existing features and their current phases. This information will be updated after every monthly release.
 
-### Config / APIs
-
-
-| Feature           | Phase      
-|-------------------|-------------------
-| [Mesh-wide Settings](https://istio.io/docs/reference/config/service-mesh.html) | Alpha
-| [Basic config validation](https://github.com/istio/istio/issues/1894)    | Alpha
-
-
-### Networking
+### Traffic Management
 
 | Feature           | Phase      
 |-------------------|-------------------
-| [Protocols HTTP 1.1](https://github.com/istio/istio/blob/master/pilot/model/service.go#L104)         | Beta
-| [Distributed Tracing Zipkin / Jaeger](https://istio.io/docs/tasks/telemetry/distributed-tracing.html)        | Beta
-| [Ingress Controller](https://istio.io/docs/tasks/traffic-management/ingress.html)         | Alpha
-| [Basic Egress](https://istio.io/docs/tasks/traffic-management/egress.html)               | Alpha
-| [Protocols HTTP 2.0](https://github.com/istio/istio/blob/master/pilot/model/service.go#L101)        | Alpha
-| [Protocols gRPC](https://github.com/istio/istio/blob/master/pilot/model/service.go#L97)     		   | Alpha
-| [Protocols TCP](https://github.com/istio/istio/blob/master/pilot/model/service.go#L107)         	   | Alpha
+| [Memquota Implementation and Integration](https://istio.io/docs/tasks/telemetry/metrics-logs.html) | Beta
+| [Protocols: HTTP 1.1](https://github.com/istio/istio/blob/master/pilot/model/service.go#L104)  | Beta
+| [Protocols: HTTP 2.0](https://github.com/istio/istio/blob/master/pilot/model/service.go#L101)  | Alpha
+| [Protocols: gRPC](https://github.com/istio/istio/blob/master/pilot/model/service.go#L97)    | Alpha
+| [Protocols: TCP](https://github.com/istio/istio/blob/master/pilot/model/service.go#L107)     | Alpha
 | [Protocols: WebSocket](https://github.com/istio/istio/blob/master/pilot/proxy/envoy/testdata/websocket-route.yaml.golden)       | Alpha
 | [Protocols: MongoDB](https://github.com/istio/istio/blob/master/pilot/model/service.go#L134)         | Alpha
-| [Basic Routing Rules](https://istio.io/docs/tasks/traffic-management/request-routing.html)      | Alpha
-| [Istioctl CLI](https://istio.io/docs/reference/commands/istioctl.html)        | Alpha
+| [Routing Rules: Retries](https://istio.io/docs/tasks/traffic-management/request-routing.html)      | Alpha
+| [Routing Rules: Timeout](https://istio.io/docs/tasks/traffic-management/request-routing.html)      | Alpha
+| [Routing Rules: Circuit Break](https://istio.io/docs/tasks/traffic-management/request-routing.html)      | Alpha
+| [Routing Rules: Header Rewrite](https://istio.io/docs/tasks/traffic-management/request-routing.html)      | Alpha
+| [Routing Rules: Traffic Splitting](https://istio.io/docs/tasks/traffic-management/request-routing.html)      | Alpha
```

Comment:
This is way too fine grained.. And its also contradicting in terms of messaging, as these features are being used heavily in production in Envoy across Lyft, Apple, Verizon, etc.

Reply:
This is an interesting point. What if a feature does not qualify as beta or stable based on the criteria that we have established but is being used in production elsewhere? Should we forgo (or loosen) our checks so we send a consistent message?

Reply:
Our label of alpha/beta/stable includes the Istio API for configuring these features, plus istio documentation / examples of how to use them, etc. If they all meet the requirements we can move them to beta, but we need to go through the list of requirements and make sure they check all the boxes.

So just because the underlying code has been used for a while doesn't mean it is automatically beta for Istio.